### DISCUSSION TITLE
Bimodality in TSS Enrichment score

### DISCUSSION THREAD

**Discussion by lucafusarbassini** [GreenleafLab/ArchR#1962] [Questions / Documentation]
Hi,
I'm analyzing data from 10x Multiome. One of our three samples shows strong bimodality in TSS enrichment, which is something I've never seen before and can't find online. What could be the reason?
All the other QC metrics are just fine.
Thank you so much,
![download](https://github.com/GreenleafLab/ArchR/assets/53112753/f46ac152-a46a-42c2-93e8-13e981f17c69)

Luca

**Comment by rcorces** [comment]
could be multiple different cell types

**Comment by lucafusarbassini** [comment]
Thank you! Yes, we checked - but both ATAC-derived clusters and RNA-derived cell types are equally distributed in the two peaks of the bimodal TSSEnrichment curve
